test/migrations/deploy-contracts-for-test.ts

```diff
@@ -186,6 +187,25 @@ export async function executeGrantsProgramProposalForTest(
   }
 }
 
+export async function executeNewAssetListingForTest(
+  deployedContracts: AllDeployedContracts,
+) {
+  const deployConfig = getDeployConfig();
+  if (config.TEST_LIST_NEW_MARKETS_WITH_PROPOSAL) {
+    await listNewMarketsViaProposal({
+      dydxTokenAddress: deployedContracts.dydxToken.address,
+      governorAddress: deployedContracts.governor.address,
+      priorityExecutorStarkware: '0xa306989BA6BcacdECCf3C0614FfF2B8C668e3CaE',
+      starkexHelperGovernor: '0x0db9b3F7Dd83e29C9bece8E5e1089bA4369E694a',
```

Comment:
Can we make this constants somewhere since they are shared across files?

Reply:
changed to be mainnetAddresses